File actions now always appear in dropdown form

PVince81 opened the pull request:
## Description
Removed the inline file action buttons, they are now always available
under the three dots button.

## Related Issue
Fixes https://github.com/owncloud/enterprise/issues/3740

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
Manual test in regular and responsive mode: the three dots button still works.

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:
<!-- Tick the checkboxes when done. -->
<!-- Raise documentation ticket in https://github.com/owncloud/documentation -->
- [x] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:
<!-- In case of incomplete PR, please list the open tasks here -->
- [x] changelog

PVince81 commented:
Fixed the test failures and squashed, the way to detect disabled actions in the dropdown is different: in the dropdown the action is simply not present instead of being a disabled button.

kulmann commented on tests/acceptance/pageObjects/FilesPageElement/filesList.js:
```diff
@@ -843,7 +803,7 @@ module.exports = {
     },
     fileActionsButtonInFileRow: {
-      selector: '//button[@aria-label="show-file-actions"]',
+      selector: '//button[@aria-label="Show file actions"]',
```
Please introduce a class and select on that instead of the aria label text

PVince81 replied:
Done

PVince81 commented:
@kulmann thanks for the review. I changed the label to "More" and added a small margin to add some distance with the scrollbar.
Also added the class.

PVince81 commented:
regression: https://github.com/owncloud/phoenix/issues/2998
